api/test/index.js

```diff
@@ -228,6 +235,432 @@ export class Test {
     )
   }
 
+  // DOM Assertions
+
+  /**
+   * Sleep for ms with an optional msg
+   *
+   * @param {number} ms
+   * @param {string} [msg]
+   * @returns {Promise<void>}
+   *
+   * @example
+   * await t.sleep(100)
+   */
+  async sleep (ms, msg) {
+    msg = msg || `Sleep for ${ms}ms`
+    await (new Promise((resolve) => setTimeout(resolve, ms)))
+    this.pass(msg)
+  }
+
+  /**
+   * Request animation frame with an optional msg
+   *
+   * @param {string} [msg]
+   * @returns {Promise<void>}
+   *
+   * @example
+   * await t.raf()
+   */
+  async raf (msg) {
```

Comment:
Rename to `requestAnimationFrame`?

Reply:
I'm find with renaming this to the full requestAnimationFrame but this one does tend to get sprinkled throughout tests as a way to trigger render frames. If it gets annoying to type out requestAnimationFrame every time maybe we add an alias to the raf name again. But fine shipping initially without it.

Reply:
i like aliases and verbose names!